docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+Throttle Commands and Modes
+***********************************************
+Vertiq modules can receive throttle commands from a variety of sources, and interpret them differently based on the module's configurations. The following section
+provides greater detail on what exactly a throttle command is and how Vertiq modules interpret them.
+
+Note that generally a module needs to be :ref:`armed <manual_advanced_arming>` in order to spin when it receives a throttle command. Refer to the :ref:`manual_advanced_arming` section 
+for more information on arming.
+
+Module Support
+===============
+The throttle parsing and modes described here apply to speed modules. While servo modules can also receive commands to spin, they are not considered 
+throttle commands for the purposes of this documentation. Controlling servos will be covered in a separate section.
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Throttle Commands and Modes
+
+	+-------------+------------------------------------+
+	| Module      | Throttle Support                   |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support the throttle parsing and modes described in this section of the documentation, documentation for servo module parsing will be provided in a
+separate section.
+
+.. table:: Servo Module Support for Throttle Commands and Modes
+
+	+-------------+------------------------------------+
+	| Module      | Throttle Support                   |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+.. _throttle_def:
+
+Throttle Definition
+==========================
+This section defines what is meant by a “throttle command” in the rest of this document. **A throttle command is a command that can come from a variety of different sources 
+that encodes a value from 100% to -100%. The magnitude of this percentage controls how quickly the module should spin, and the sign indicates the direction of the spin.**
+
+The exact drive voltage/speed and direction that a throttle command commands depends on how the module is configured by the user. A throttle command of 100% will command 
+the module to spin at 100% of whatever the maximum set by its mode is, and in the direction set by its direction configuration. 

```

Comment:
A throttle command of 100% commands the module to spin at 100% of its mode's maximum in the direction of its direction configuration. 
Same for the -100 sentence